plugin/cache/metrics.go

```diff
@@ -0,0 +1,52 @@
+package cache
+
+import (
+	"github.com/coredns/coredns/plugin"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	// cacheSize is total elements in the cache by cache type.
+	cacheSize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: plugin.Namespace,
+		Subsystem: "cache",
+		Name:      "size",
```

Comment:
"size" is ambiguous, Size of what? I think you meant "entries" to match the similar metric in dnssec.

```suggestion
		Name:      "entries",
```

Reply:
These metrics are split from `plugin/cache/handler.go`, could these modifications be placed in the next pr?

Reply:
Sure, makes sense.